examples/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.js

```diff
@@ -0,0 +1,36 @@
+// Import the NLCD NALCMS collection.
+var dataset = ee.Image('USGS/NLCD_RELEASES/2019_REL/NALCMS/2020');
+
+// Filter the collection to the 2019 product.
+var nlcd2019 = dataset.filter(ee.Filter.eq('system:index', '2019')).first();
+
+// Each product has multiple bands for different rangeland categories.
+print('Bands:', nlcd2019.bandNames());
+
+// Select the landcover band.
+var landcover = nlcd2019.select('landcover units_count');
+
+var vis = {

```

Comment:
Only 19 classes, so something simpler will work.  These are randomly chosen colors.  Feel free to change them!

```JavaScript
Map.addLayer(image, {
  palette: [
    'tomato',  // 1
    'orange',  // 2
    'red',  // 3
    'purple',  // 4
    'maroon',  // 5
    'silver',  // 6
    'blue',  // 7
    'coral',  // 8
    'cyan',  // 9
    'bisque',  // 10
    'darkviolet',  // 11
    'goldenrod',  // 12
    'lightsalmon',  // 13
    'fuchsia',  // 14
    'seagreen',  // 15
    'whitesmoke',  // 16
    'thistle',  // 17
    'sandybrown',  // 18
    'seashell',  // 19
    ],
  min:1, max:19});

Map.setCenter(-114, 38, 6);
```

Reply:
Updated